frontend/src/pages/Room.jsx

```diff
@@ -1,41 +1,52 @@
-import useAuth from '../hooks/useAuth'
-import { useSearchParams } from 'react-router-dom'
+import { useNavigate, useParams } from 'react-router-dom'
 import axios from '../api/axios'
 import { useEffect, useState } from 'react'
 import { URL_MATCHING_ROOM, STATUS_CODE_SUCCESS } from '../constants'
+import QuestionPane from '../components/QuestionPane'
+import Editor from '../components/collaboration/Editor'
+import { Button } from '../components/Button'
 
 const Room = () => {
-  const { auth } = useAuth()
-  const [questionId, setQuestionId] = useState('')
-  let [searchParams] = useSearchParams()
+  const navigate = useNavigate()
+  const { roomId } = useParams()
+  const [questionId, setQuestionId] = useState()
 
   useEffect(() => {
     getQuestionId()
   })
 
   const getQuestionId = async () => {
-    const res = await axios
-      .get(`${URL_MATCHING_ROOM}/${searchParams.get('roomId')}`)
-      .catch((e) => {
-        console.log(e)
-      })
+    const res = await axios.get(`${URL_MATCHING_ROOM}/${roomId}`).catch((e) => {
+      console.log(e)
+    })
     if (res && res.status === STATUS_CODE_SUCCESS) {
       setQuestionId(res.data.data.questionId)
     }
   }
 
+  const endSession = async () => {
+    await axios.delete(`${URL_MATCHING_ROOM}/${roomId}`).catch(console.log)
+    alert('End session! Going to Home...')
```

Comment:
Perhaps can consider replacing alert with alert/toast/modal from Chakra moving forward, for sake of consistency
https://chakra-ui.com/docs/components/alert 
https://chakra-ui.com/docs/components/toast

Reply:
Thanks for the suggestion, replaced with Chakra toast.